### ISSUE TITLE
Can't add devices after opening nr-gab and logging in : 

### ISSUE THREAD

**Issue by ewidance** [hardillb/node-red-contrib-googlehome#19]
I wad using nr-gab and node-red-contrib-googlehome for a while and made some modifications on my devices. Since thery where not properly enumerated on Google Home App, i removed "NR-GAB" and try to reassociate.

Atfter loggin in, (aparently successfully), Google home refuse to enumerate and add devices. Status line displays : 

- NR-GAB associated
- An error occured, please retry

I've retried 3/4 times during the day with no changes.

Have i missed something?

**Comment by hardillb** [comment]
There is nothing showing as wrong with the system

What devices did you change and how?

I suggest you delete the devices you edited and then sync again. Once it' sync's clean recreate the devices one at a time.

**Comment by ewidance** [comment]
I added a  thermostat device and removed an other one, (to change name).  However i could not find a way to removed the old one in Google Home that's why I disconnect account and try to reconnect. 

I just tested to remove the added thermostat, and same thing.

Here's my current device list.
![2022-08-07 22_39_47-Node-RED Google Assistant Bridge](https://user-images.githubusercontent.com/2135006/183310236-1208b3c0-a6d5-4e0d-a75a-7a3c5ef6f68a.png)
